Linked Accounts/Link-Accounts.ps1

```diff
@@ -613,98 +528,94 @@ Write-LogMessage -Type Info -MSG "Getting PVWA Credentials to start Linking acco
 $caption = "Account Link Utility"
 $msg = "Enter your User name and Password";
 $creds = $Host.UI.PromptForCredential($caption, $msg, "", "")
-if ($null -ne $creds)
-{
-	if ($AuthType -eq "radius" -and ![string]::IsNullOrEmpty($OTP))
-	{
+if ($null -ne $creds) {
+	if ($AuthType -eq "radius" -and ![string]::IsNullOrEmpty($OTP)) {
 		$g_LogonHeader = $(Get-LogonHeader -Credentials $creds -RadiusOTP $OTP)
-	}
-	else
-	{
+	} else {
 		$g_LogonHeader = $(Get-LogonHeader -Credentials $creds )
 	}
-}
-else
-{
+} else {
 	Write-LogMessage -Type Error -MSG "No Credentials were entered" -Footer
 	exit
 }
 #endregion
 
+If (Compare-Version("11.7")) {$extraPass = "extraPasswordIndex"} else {$extraPass = "extraPasswordID"}
+
 #region [Read Accounts CSV file and link Accounts]
-If ([string]::IsNullOrEmpty($CsvPath))
-{
+If ([string]::IsNullOrEmpty($CsvPath)) {
 	$CsvPath = Open-FileDialog($g_CsvDefaultPath)
 }
 $delimiter = $((Get-Culture).TextInfo.ListSeparator)
 $accountsCSV = Import-Csv $csvPath -Delimiter $delimiter
-$rowCount = $accountsCSV.Count
-$counterMaster = 0
-Write-LogMessage -Type Info -MSG "Starting to add links to $rowCount master accounts" -SubHeader
+
+$ExtraPass1Count = @($accountsCSV |Where-Object ExtraPass1Name -ne "" ).count
+$ExtraPass2Count = @($accountsCSV |Where-Object ExtraPass2Name -ne "" ).count
+$ExtraPass3Count = @($accountsCSV |Where-Object ExtraPass3Name -ne "" ).count
+$linkCount =  $ExtraPass1Count + $ExtraPass2Count + $ExtraPass3Count
+
+$masterCount = @($accountsCSV | Select-Object -Property userName,address -Unique).Count
```

Comment:
Same here, what do you need this counter for?

Reply:
It's used on line 631